knative-operator/pkg/apis/operator/v1alpha1/knativekafka_lifecycle.go

```diff
@@ -0,0 +1,31 @@
+package v1alpha1
+
+import "knative.dev/pkg/apis"
+import knativeoperatorv1alpha1 "knative.dev/operator/pkg/apis/operator/v1alpha1"
+
+var (
+	kafkaCondSet = apis.NewLivingConditionSet(
+		knativeoperatorv1alpha1.DependenciesInstalled,
```

Comment:
What would this be in the Kafka context?

Reply:
good point. will get rid of this.
was a simple copy paste

Reply:
done